WalletWasabi.Fluent/ViewModels/Wallets/Send/TransactionPreviewViewModel.cs

```diff
@@ -266,7 +266,7 @@ private async Task<bool> InitialiseTransactionAsync()
 
 			if (!result)
 			{
-				await ShowErrorAsync("Transaction Building", "At the moment, it is not possible to select a transaction fee that is less than the payment amount. The transaction cannot be sent.",
+				await ShowErrorAsync("Transaction Building", "The transaction cannot be sent. It is not possible to select a transaction fee that is more than the payment amount.",
```

Comment:
How about a shorter and a more precise message:

```suggestion
				await ShowErrorAsync("Transaction Building", "The transaction cannot be sent because its fee is more than the payment amount.",
```